playbooks/roles/crc/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# setup crc
+- name: "Set crc memory"
+  command: "crc config set memory {{ crc_memory | default('30156') }}"
```

Comment:
shouldnt we use the crc binary path for this step ? 
or use shell module in ansible ?

Reply:
Because you have to [manually extract](https://crc.dev/crc/#installing_gsg/) CRC, we don't have a default value for the location of the crc binary.  The `command` module does reference $PATH though, so it will still work.  I can change to shell, but from what I can tell the only difference is that `shell` supports input/output redirection